LadybugTools_Engine/Python/src/ladybugtools_toolkit/external_comfort/_externalcomfortbase.py

```diff
@@ -12,8 +12,9 @@
 from ladybug_comfort.collection.pmv import PMV
 from ladybug_comfort.collection.utci import UTCI
 from matplotlib.figure import Figure
+from matplotlib.colors import LinearSegmentedColormap
 
-from ..bhom.logging import CONSOLE_LOGGER
+from python_toolkit.bhom.logging import CONSOLE_LOGGER
```

Comment:
```suggestion
from ..bhom.logging import CONSOLE_LOGGER
```
This is why the logger is saying Python Toolkit rather than LadybugTools_Toolkit - this may have been my fault when rebasing 🙈